bbq_manager/urls.py

```diff
@@ -11,4 +11,5 @@
     path('api/auth/jwt', include('djoser.urls.jwt')),
     path('api/schema/', SpectacularAPIView.as_view(), name='schema'),
     path('api/docs', SpectacularSwaggerView.as_view(url_name='schema'), name='docs'),
+    path('api/company', include('company.urls')),
```

Comment:
Урлы должны заканчиваться слешем(канон django). 
```suggestion
    path('api/companies/', include('company.urls')),
```